pytrendy/post_processing/segments_refine/artifact_cleanup.py

```
@@ -259,7 +259,7 @@ def has_partial_overlap_prev(segment: dict, segment_prev: dict) -> bool:
         is_flat = segment['direction'] == 'Flat'
         is_gradual = ('trend_class' in segment and segment['trend_class'] == 'gradual')
         is_abrupt = ('trend_class' in segment and segment['trend_class'] == 'abrupt')
-        is_padded = is_abrupt and ('padded' in segment) and (segment['padded'] == True)
+        is_padded = ('padded' in segment) and (segment['padded'] == True)
```

Ruff (0.15.21)

[error] Avoid equality comparisons to `True`; use `segment['padded']:` for truth checks

Replace with `segment['padded']`

(E712)